frameworks/g-cloud-12/questions/services/systemRequirements.yml

```diff
@@ -16,10 +16,10 @@ type: list
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter a system requirement.
   - name: max_items_limit
     message: You can have no more than 10 system requirements.
   - name: under_10_words
-    message: You can’t write more than 10 words for each system requirement.
+    message: Each system requirement must be 10 words or fewer.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?